web/src/components/filings-calendar/FilingsCalendarSingleGrid.tsx

```diff
@@ -88,6 +99,17 @@ export const FilingsCalendarSingleGrid = (props: Props): ReactElement => {
           />
         );
       }
+
+      if (event.calendarEventType === "XRAY") {
```

Comment:
pebble - some parts of your code is generic like [renewalCalendarEventUrlSlug], but in other parts it's specific like calendarEventType === "XRAY". I think it's better to be consistent.